README.md

```diff
@@ -1,2 +1,113 @@
-# go-chpr-logger
-Go logging with chpr conventions
+➤ Readme generated with godocdown
+
+This package implements our standard Logrus + Logentries + Sentry configuration.
+
+The package manages a singleton instance of logrus.Logger, initialized from
+environment variables.
+
+You will use it exactly as you would use logrus.
+
+
+### Examples
+
+Log an info:
+
+    WithFields(logger.Fields{
+    	"driverId": driverId,
+    	"rideId": rideId,
+    }).Info("Ride accepted")
+
+Log an info with a single field:
+
+    WithField("userId", userId).Info("User logged in")
+
+Log an error:
+
+    WithFields(logger.Fields{
+    	"event": event,
+    	"err": err
+    }).Error("Could not store ride end event")
+
+IMPORTANT: when logging an error or a warning, don't put an error you're not
```

Comment:
you are talking about the message of the `err` error ? I had to read this warning twice which may be a sign it can made be clearer, or that I'm too tired ;)